spec/controllers/invitations_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe InvitationsController, type: :controller do
+  describe 'GET #new' do
+    it 'returns http success' do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET #show' do
```

Comment:
This test is also failing, why? Dig deep into it to find out why. Tests are a good way to check your understanding of Rails. I'd suggest looking at your routes with `$ rails routes` and checking to see if everything makes sense in there. For example, you don't really have a `show` controller for Invitations, but you have a route for it. The show route is usually `/invitations/show/:id` but you have a created an extra one with `invitations/show`, why? I think this is an indication of a gap in your understanding of routes and controllers.

[`REQUIRED`]